lucille-plugins/lucille-tika/src/main/java/com/kmwllc/lucille/tika/stage/TextExtractor.java

```diff
@@ -103,14 +122,31 @@ public Iterator<Document> processDocument(Document doc) throws StageException {
       parseInputStream(doc, inputStream);
 
     } else if (doc.has(filePathField)) {
-
+      // get fileObject from path
       String filePath = doc.getString(filePathField);
+      FileObject file = FileUtils.getFileObject(filePath, fsManager);
+
+      // if file null, don't process document
+      if (file == null) {
+        return null;
+      }
+
+      // getting inputStream and catching errors gracefully
+      InputStream inputStream;
+      try {
+        inputStream = file.getContent().getInputStream();
```

Comment:
suggest adding comment that file.getContent() never returns null; can include link to vfs doc where it says this